src/main/resources/application.properties

```diff
@@ -1 +1,6 @@
+spring.datasource.url=jdbc:postgresql://localhost:5433/university
```

Comment:
create class DatabaseProperties for url, username and password, use:
```
@ConfigurationProperties(prefix = "yourPrefix")
@Configuration
```
use this env variables in application.properties